1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,55 @@
-function setAlarm() {}
+const getTimeString = ({ hours, minutes, seconds, zone }) => {
+  if (minutes / 10 < 1) {
+    minutes = "0" + minutes;
+  }
```

Comment:
be careful with your types here; `minutes` is a `Number`, so doing division on it is fine, but concatenating it to a string without casting it explicitly to a string first (e.g. `minutes.toString()`) can be confusing